docs/markdown/Helm/helm-deployments.md

```diff
@@ -182,6 +182,33 @@ We believe that this approach gives a very consistent and predictable ordering w
 
 In addition to value files, you can also use inline values in your `helm_deployment` targets by means of the `values` field. All inlines values that are set this way will override any entry that may come from value files.
 
+Third party chart artifacts
+---------------------------
+
+Previous examples on the usage of the `helm_deployment` target are all based on the fact that the deployment declares a dependency on a Helm chart that is also part of the same repository. Since charts support having dependencies with other charts in the same repository or with external 3rd party Helm artifacts (declared as `helm_artifact`), all that dependency resolution is handled for us.
+
+However, `helm_deployment`s are not limited to only first party charts, as it is also possible to declare a deployment having a dependency on a 3rd party Helm artifact instead. As an example, consider the following workspace layout:
+
+```python 3rdparty/helm/jetstack/BUILD
+helm_artifact(
+  name="cert-manager",
+  artifact="cert-manager",
+  version="v0.7.0",
+  repository="https://charts.jetstack.io",
+)
+```
+```python src/deploy/BUILD
+helm_deployment(
+  name="main",
+  dependencies=["//3rdparty/helm/jetstack:cert-manager"],
```

Comment:
How hard would it be to add dependency inference for helm_artifacts?

Reply:
In the deployments is not possible to infer the chart, first or third party. Deployments are basically a set of YAML files and settings with no info on what chart they are meant to be used on.